c++/include/orc/Writer.hh

```diff
@@ -262,6 +262,19 @@ namespace orc {
      * @return if not set, the default is false
      */
     bool getUseTightNumericVector() const;
+
+    /**
+     * Set the initial buffer capacity of output stream.
+     * Each column contains multiple output streams with buffers,
+     * and these buffers will automatically expand when memory is exhausted.
```

Comment:
```suggestion
     * and these buffers will automatically expand when necessary but up to the compression block size.
```

Reply:
When the used size of the buffer reaches the initial capacity, the buffer will continue to expand in compression block size units.

Reply:
In the `CompressionStream`, will the output buffer exceed the compression block size?

Reply:
The outputBuffer(char* variable) of the class CompressionStreamBase doesn't exceed the compression block size,  and the dataBuffer(BlockBuffer variable) of the class BufferedOutputStream will exceed the compression block size.